WcaOnRails/db/migrate/20190728084145_add_wrc_fields_to_delegate_reports.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddWrcFieldsToDelegateReports < ActiveRecord::Migration[5.2]
+  def change
+    add_column :delegate_reports, :wrc_feedback_requested, :boolean, default: false
```

Comment:
Should this also have `null: false`? I think that as written here, this will allow for 3 values to `wrc_feedback_request`: `true, false, null`. We generally don't want tri-state booleans like this.

Reply:
Good catch! Totally agree - we don't want to accept null values.